### ISSUE TITLE
DashboardScene: add unit tests for keybindings

### ISSUE THREAD

**Issue by Sergej-Vlasov** [grafana/grafana#89940]
add unit tests for `public/app/features/dashboard-scene/scene/keyboardShortcuts.ts` for editor and viewer users

**Comment by devesh-2002** [comment]
May I take this issue?

**Comment by Sergej-Vlasov** [comment]
> May I take this issue?

HI @devesh-2002! Yes, indeed, contributions are always warmly welcome! Probably best to add tests in `public/app/features/dashboard-scene/scene/DashboardScene.test.tsx` to save time on test setup, but feel free to come up with a different approach!

**Comment by AlejandroBlanco2001** [comment]
Hey @Sergej-Vlasov, is this issue already addressed? I'd like to take it if it's available 😄.

**Comment by Sergej-Vlasov** [comment]
@AlejandroBlanco2001 I don't think this was addressed. Feel free to assign yourself to this issue and give it a go!

**Comment by AlejandroBlanco2001** [comment]
Hey @Sergej-Vlasov , I understand most of the code involved, but I see your point that setting up tests for the key bindings is quite difficult. Is it possible to have some logs or messages that go through the codebase, or any similar test?

**Comment by anglerfishlyy** [comment]
Hello @Sergej-Vlasov ,  Would it be alright if I pick this up and work on a fresh PR?

**Comment by Sergej-Vlasov** [comment]
Hi @anglerfishlyy ! Seems like there is already a unit test PR for this https://github.com/grafana/grafana/pull/107065. Haven't looked at it yet, but if you have a better approach then feel free to go ahead! As an alternative you could write some E2E playwright tests to have more complete coverage

**Comment by AritraDey-Dev** [comment]
@Sergej-Vlasov #107065  got closed due to no review. Can you PTAL?
